static/bidder-info/seedingAlliance.yaml

```diff
@@ -0,0 +1,12 @@
+maintainer:
+  email: tech@seeding-alliance.de
+gvlVendorID: 371
+capabilities:
+  site:
+    mediaTypes:
+      - native
+      - banner
+userSync:
+  redirect:
+    url: https://dmp.nativendo.de/set-uuid?gdpr={{.GDPR}}&gdpr_consent={{.GDPRConsent}}&redirect_url={{.RedirectURL}}
+    userMacro: $UID
```

Comment:
Please make sure this URL is correct and returns redirect with user macro. 
I tested it with encoded Prebid Server url (https://prebid.adnxs.com/pbs/v1/setuid?bidder=3DseedingAlliance&gdpr=&gdpr_consent=&f=i&uid=$UID): `https://dmp.nativendo.de/set-uuid?gdpr=&gdpr_consent=&redirect_url=https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3DseedingAlliance%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%24UID` and expect to see redirect, but see  http 400 error code instead. 

Example of a working URL https://js.adscale.de/pbsync.html?gdpr=&gdpr_consent=&redirect=https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3Dbliink%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D
It returns http 200 and then http 302 statuses

Reply:
The URL itself is fine but we expect a non nil value for **gdpr** on our end.

Meaning that the following URL should work fine:
`https://dmp.nativendo.de/set-uuid?gdpr=0&gdpr_consent=&redirect_url=https://prebid.adnxs.com/pbs/v1/setuid?bidder=seedingAlliance&gdpr=&gdpr_consent=&f=i&uid=$UID`

Is this an issue?

Reply:
Oh I see, please make sure you support encoded redirect URL, this is how PBS will send it.

Reply:
Just made sure that we actually support encoded redirect URLs. 👍

Reply:
We cannot merge this PR if User sync URL doesn't work. It should work with encoded redirect URL and return http 200 and http 302 statuses.

Reply:
https://dmp.nativendo.de/set-uuid?gdpr=0&gdpr_consent=&redirect_url=https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3DseedingAlliance%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%24UID

That 400 response is not coming from our DMP. That status (incl. the following error message: `The bidder name provided is not supported by Prebid Server`) shows up after the redirect since I guess the adapter isn't merged yet.

Reply:
@sag-celejd The user sync url fails if the gdpr field is empty. Prebid Server will set the `gdpr` parameter to either empty, 0, or 1 and all must be supported. An empty gdpr is a lack of a signal versus an explicit 0 or 1 signal. If the gdpr is empty we receive a 400 and thats is what @VeronikaSolovei9 is referring to.

Reply:
Alright, I just changed the behaviour on our end:

https://dmp.nativendo.de/set-uuid?gdpr=&gdpr_consent=&redirect_url=https%3A%2F%2Fprebid.adnxs.com%2Fpbs%2Fv1%2Fsetuid%3Fbidder%3DseedingAlliance%26gdpr%3D%26gdpr_consent%3D%26f%3Di%26uid%3D%24UID

Should work fine now.

Reply:
Yes, works good now, thank you for addressing this!